ui/pages/home/home.component.js

```diff
@@ -636,6 +642,30 @@ export default class Home extends PureComponent {
               defaultActiveTabName={defaultHomeActiveTabName}
               onTabClick={onTabClick}
               tabsClassName="home__tabs"
+              subHeader={
+                <div
+                  className="home__subheader-link"
+                  onClick={async () => {
+                    if (permissionsSubjects[METALABS_URL]) {
+                      window.open(METALABS_URL);
+                    } else {
+                      const id = await requestAccountsPermissionWithId(
+                        METALABS_URL,
+                      );
+                      history.push(
+                        `${CONNECT_ROUTE}/${id}?redirect=${encodeURIComponent(
+                          METALABS_URL,
+                        )}`,
+                      );
+                    }
+                  }}
+                >
+                  <IconChart />
+                  <div className="home__subheader-link__text">
```

Comment:
I don't think that BEM conventions use `__` twice. The structure should be: `.block__elem--mod`